pkg/metrics/metrics.go

```diff
@@ -50,3 +50,22 @@ var (
 		}).Inc()
 	}
 )
+
+// The scheduler related metrics.
+var (
+	// SchedulingCycleDurationMilliseconds is a Fleet scheduler metric that tracks how long it
+	// takes to complete a scheduling loop run.
+	SchedulingCycleDurationMilliseconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "scheduling_cycle_duration_milliseconds",
+			Help: "The duration of a scheduling cycle run in milliseconds",
+			Buckets: []float64{
+				10, 50, 100, 500, 1000, 5000, 10000, 50000,
```

Comment:
any reason to use the this bucket size?

Reply:
Oh, the unit is milliseconds and based on prev. observations these buckets seem to be appropriate => would you prefer finer bucket sizes?

Reply:
how about copy controller-runtime one?

Reply:
Oh, the controller-runtime one uses custom ExponetialBuckets (`prometheus.ExponentialBuckets(10e-9, 10, 10)`) and the unit is in seconds -> I can send a separate PR if you'd like to switch to seconds as well